### PR TITLE
Add Edge (chromium) support

### PR DESCRIPTION
**Pull request by 810**
no test

Pull Request for Issue # .

### Summary of Changes

add support new edge

### Testing Instructions

code review

### CODE REVIEW

**Comment by mbabker** [comment]
For completeness, https://github.com/joomla-framework/application/pull/86 added the same for the `Joomla\Application\Web\WebClient` class and I went ahead and tagged a new release with that included.  So if someone wants to do the `composer update` then both browser detection libraries will have this addition once this PR and the upstream change are applied.

File: libraries/src/Environment/Browser.php [hunk 1]
```diff
@@ -578,6 +578,15 @@ public function match($userAgent = null, $accept = null)
 					$this->minorVersion = 0;
 				}
 			}
+			/*
+			 * We have to check for Edge as the first browser, because Edge has something like:
+			 * Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/75.0.3738.0 Safari/537.36 Edg/75.0.107.0
+			 */
+			elseif (preg_match('|Edg\/([0-9.]+)|', $this->agent, $version))
+			{
+				$this->setBrowser('edg');
```
Comment (Quy) [inline libraries/src/Environment/Browser.php:587]:
```suggestion
				$this->setBrowser('edg');

```

**Comment by 810** [comment]
fixed the codestyle

**Comment by Quy** [comment]
How about using existing code to handle the new version as follows:

```
			if (preg_match('|Edg(e)?\/([0-9.]+)|', $this->agent, $version))
			{
				$this->setBrowser('edg' . $version[1]);

				if (strpos($version[2], '.') !== false)
				{
					list($this->majorVersion, $this->minorVersion) = explode('.', $version[2]);
				}
				else
				{
					$this->majorVersion = $version[2];
					$this->minorVersion = 0;
				}
			}
```

**Comment by 810** [comment]
`if (preg_match('|Edg(e)?\/([0-9.]+)|', $this->agent, $version))`

Edge will stay at 2 different versions <75 Will be the edge html, and above 75 will use the blink html.

**Comment by Quy** [comment]
`$this->setBrowser('edg' . $version[1]);`

It will be set to `edge` or `edg` accordingly.

**Comment by HLeithner** [comment]
thx

**Pull request merged**